src/tests/testFtpEpsv.cc

```diff
@@ -0,0 +1,88 @@
+/*
+ * Copyright (C) 1996-2025 The Squid Software Foundation and contributors
+ *
+ * Squid software is distributed under GPLv2+ license and includes
+ * contributions from numerous individuals and organizations.
+ * Please see the COPYING and CONTRIBUTORS files for details.
+ */
+
+#include "squid.h"
+#include "compat/cppunit.h"
+#include "unitTestMain.h"
+
+#include "clients/FtpClient.h"
+#include "sbuf/SBuf.h"
+
+class TestFtpEpsv : public CPPUNIT_NS::TestFixture
+{
+    CPPUNIT_TEST_SUITE(TestFtpEpsv);
+    CPPUNIT_TEST(testValidTupleParses);
+    CPPUNIT_TEST(testRejectDigitDelimiter);
+    CPPUNIT_TEST(testPortRange);
+    CPPUNIT_TEST(testTrailingGarbage);
+    CPPUNIT_TEST(testOptionalWsAllowed);
+    CPPUNIT_TEST_SUITE_END();
+
+protected:
+    void testValidTupleParses();
+    void testRejectDigitDelimiter();
+    void testPortRange();
+    void testTrailingGarbage();
+    void testOptionalWsAllowed();
+};
+
+CPPUNIT_TEST_SUITE_REGISTRATION(TestFtpEpsv);
+
+void
+TestFtpEpsv::testValidTupleParses()
+{
+    uint16_t port = 0;
+
+    // Classic '|' delimiter
+    CPPUNIT_ASSERT(Ftp::parseEPSV(SBuf("(|||12345|)"), port));
```

Comment:
Current build failure is due to missing `Tokenizer` (first parameter) on all these test calls of `Ftp::parseEPSV`
```
src/tests/testFtpEpsv.cc:42:60: error: too few arguments to function call, expected 3, have 2
   42 |     CPPUNIT_ASSERT(Ftp::parseEPSV(SBuf("(|||12345|)"), port));
      |                    ~~~~~~~~~~~~~~                          ^
```